src/main/topLevelPages/myDashboard/MyDashboardWithoutMemberships.tsx

```diff
@@ -18,13 +18,13 @@ const MyDashboardWithoutMemberships: FC<MyDashboardWithoutMembershipsProps> = ({
   const columns = useColumns();
   return (
     <>
-      <PageContentColumn columns={8}>
-        <MembershipSuggestions />
-        <StartingSpace />
+      <PageContentColumn columns={12}>
```

Comment:
We should revise the positioning of the blocks as in some cases the layout looks a bit broken - having the Explore section full-width and the other 4-col widgets below it all one above the other on the left.